teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization, _name = 'unknown')
+    super('Mary', 17)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Also here in the `Teacher` constructor you should send the data for the `Person` class constructor as it is its parent class.

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
end
```